src/applications/simple-forms/mock-simple-forms-patterns/pages/mockTextInputSsn.js

```diff
@@ -5,16 +5,11 @@ import {
   inlineTitleUI,
   inlineTitleSchema,
 } from 'platform/forms-system/src/js/web-component-patterns';
-import { ssnUI } from 'applications/caregivers/definitions/UIDefinitions/sharedUI';
 
 /** @type {PageSchema} */
 export default {
   uiSchema: {
     ...titleUI('RJSF'),
-    ssn: {
-      ...ssnUI(),
-      'ui:title': 'Social security number',
-    },
     vaFileNumber: {
```

Comment:
could you also remove vaFileNumber and 'view:wcv3Title' here and update titleUI? Typically we keep ssn and vaFileNumber together, so might as well remove both

Reply:
Done!